api/src/main/java/uk/gov/dhsc/htbhf/claimant/service/claim/ClaimService.java

```diff
@@ -80,12 +82,25 @@ private void sendMessagesForNewClaim(EligibilityAndEntitlementDecision decision,
         if (identityAndEligibilityResponse.isEmailAndPhoneMatched()) {
             claimMessageSender.sendInstantSuccessEmailMessage(claim, decision);
         } else {
-            claimMessageSender.sendDecisionPendingEmailMessage(claim);
+            sendMessagesForPhoneOrEmailMismatch(identityAndEligibilityResponse, claim);
         }
         claimMessageSender.sendNewCardMessage(claim, decision);
         claimMessageSender.sendReportClaimMessage(claim, identityAndEligibilityResponse, ClaimAction.NEW);
     }
 
+    private void sendMessagesForPhoneOrEmailMismatch(CombinedIdentityAndEligibilityResponse identityAndEligibilityResponse, Claim claim) {
+        claimMessageSender.sendDecisionPendingEmailMessage(claim);
+
+        List<LocalDate> childrenDobFromBenefitAgency = identityAndEligibilityResponse.getDobOfChildrenUnder4();
+        List<LocalDate> declaredChildrenDob = claim.getClaimant().getInitiallyDeclaredChildrenDob();
+
+        if (childrenDobFromBenefitAgency.containsAll(declaredChildrenDob)) {
```

Comment:
This doesn't cater for there being more children returned from DWP than they have declared.

Reply:
We said in that scenario we'll still return children match, there is a unit test covering that scenario. @YetAnotherMatt can you confirm that's correct?

Reply:
That is correct, yes. DWP reporting extra children is ok.